src/gui/folderwatcher.h

```diff
@@ -88,6 +88,11 @@ class FolderWatcher : public QObject
      *  of the contained files is changed. */
     void pathChanged(const QString &path);
 
+    /*
+    * Emitted when office lock files were removed
+    */
+    void officeFilesLockReleased(const QSet<QString> &files);
```

Comment:
I would not use `office` in the name as we may want to have it more generic than just some office files